chunks/lang-node/Dockerfile

```diff
@@ -9,12 +9,15 @@ USER gitpod
 ENV TRIGGER_REBUILD=1
 ENV NODE_VERSION=${NODE_VERSION}
 
+ENV PNPM_HOME=/home/gitpod/.pnpm
+ENV PATH=/home/gitpod/.nvm/versions/node/v${NODE_VERSION}/bin:/home/gitpod/.yarn/bin:${PNPM_HOME}:$PATH
```

Comment:
Adding `PNPM_HOME` to the path is great, and fixes #913, and adding `/home/gitpod/.yarn/bin` fixes #639 , thank you @gengjiawen !

Reply:
Yeah，I have put it pr description ，when merged should close related issues

Reply:
Indeed :) I was just sharing praise, but also highlighting the part of this PR that resolves the context issues.